### ISSUE TITLE
[BUG] Bot is not understanding valid commands

### ISSUE THREAD

**Issue by ReinitiaIized** [ErdbeerbaerLP/Discord-Chat-Integration#46]
**What happened?**
https://cdn.discordapp.com/attachments/684977927732854785/692866169501384724/unknown.png

Despite using a valid command, bot is saying the command is unknown

**What should happen?**

Bot executes the command as expected

**Steps to reproduce**
See "What Happened"

**Version**
1.12.2

Mod version 1.1.13

**Comment by ReinitiaIized** [comment]
Additionally:
- it's not editing the channel description as intended (it does have permission)
https://cdn.discordapp.com/attachments/684977927732854785/692867982329643008/unknown.png
- it doesn't properly recognize the amount of Players in-game
https://cdn.discordapp.com/attachments/684977927732854785/692868051334332521/unknown.png
- it's not replicating in-game chat to Discord, but Discord to in-game is working fine

**Comment by ReinitiaIized** [comment]
Downgrading to mod version 1.1.12 fixed all above problems, with the exception of the bot not reporting player numbers correctly in it's playing status
https://cdn.discordapp.com/attachments/684977927732854785/692880379027259422/unknown.png

**Comment by ReinitiaIized** [comment]
So this issue popped up again on 1.1.12. Looked into logs, nothing which caught the eye. If you have anything specific you want me to do to help find the problem, I'll provide the relevant information.

**Comment by ErdbeerbaerLP** [comment]
Can you send me your config (without the bot token!)? This looks like an config issue for me

**Comment by ReinitiaIized** [comment]
Here's config. Do note: After downgrading to 1.1.12 and setting up a new config file, the mod was functional for a few days. It just suddenly stopped working, and nothing was changed in this config file
https://pastebin.com/raw/uKRd5iNL

**Comment by ReinitiaIized** [comment]
Was able to discover the problem: Discord Permissions.
I've been attempting to set only the permissions the bot needs, and didn't realize I temporarily gave the role Administrator. As soon as I removed it, the bot gave me a hint regarding not have the correct permissions (message read, send, manage, manage role, embed link). I gave the bot all those permissions on the desired channel, still no go. As soon as I gave it Administrator and rebooted, everything worked as intended.

Essentially, the checks regarding permissions are apparently not working correctly, as it has the right permissions but still doesn't seem to think it does despite not displaying an error message.
